conferences/jsconf.yaml

```diff
@@ -21,6 +21,13 @@
 #     status: opencfp # and add onsale to the same line for both
 #
 
+# JSConf India
```

Comment:
Wong comment

Reply:
🤦‍♀️
thanks for pointing out,
Have a fix here https://github.com/jsconf/jsconf.com/pull/54 🙏